src/bun.js/event_loop.zig

```diff
@@ -1882,6 +1884,70 @@ pub const MiniEventLoop = struct {
 
         this.loop.wakeup();
     }
+
+    pub fn stderr(this: *MiniEventLoop) *JSC.WebCore.Blob.Store {
+        return this.stderr_store orelse brk: {
+            const store = bun.default_allocator.create(JSC.WebCore.Blob.Store) catch unreachable;
+            var mode: bun.Mode = 0;
+            const fd = if (Environment.isWindows) bun.FDImpl.fromUV(2).encode() else bun.STDERR_FD;
+
+            switch (bun.sys.fstat(fd)) {
+                .result => |stat| {
+                    mode = @intCast(stat.mode);
+                },
+                .err => {},
```

Comment:
Is the .err needed if nothing is done?
```suggestion
```

Reply:
`else => {}` or `.err => {}` is needed otherwise the compiler will complain about non-exhaustive switching over the tagged union